packages/MarkdownEditor-Core.package/MarkdownCommentBlock.class/class/matches..st

```diff
@@ -0,0 +1,7 @@
+testing
+matches: aString
+	
+	^ (aString beginsWith: '    ') not
+		and: [aString withBlanksTrimmed beginsWith: '<!--']
+		and: [((aString includesSubstring: '-->') not)
+			or: [(aString withBlanksTrimmed findString: '-->') >= 5]]
```

Comment:
Line 6 and 7 are actually an implication. So we could use `==>` instead of the combination of `not` and `or:`. What do you think?